Copy to clipboard

AndrewRadev opened the pull request:
Fixed and improved version of https://github.com/AndrewRadev/vim-fmi/pull/4

aoantov committed:
clipboard feature

AndrewRadev committed:
Fixed and improved clipboard functionality

AndrewRadev committed:
Minor whitespace tweaks

AndrewRadev commented on app/javascript/application.js:
```diff
@@ -27,6 +28,21 @@ $(document).ready(function() {
+    setTimeout(() => {
+      $button.find('img').attr('src', copySrc);
+    }, 1000);
+  });
```
The code picks up "this" which is the element we targeted with `.clipboard-btn`, the button. In the button, there's an `img` tag which has the "copy" icon's image URL. We also need a "success" icon, which I've attached as a `data-success` attribute. 

Once we copy the token's text, we switch the src attribute of the image to a ✅ and after 1 second, we switch it back.

It's not the only way to do it, but it works.

Atira12 replied:
Maybe switch to 2000 and use image_tag resize to resize the copy.png because it is misaligned a little

AndrewRadev replied:
Images and emoji are difficult to mix, since emoji is text and is affected by it, and it varies across systems. To ensure consistency between the two buttons, I've changed the copy button to be a link as well and put 📋 as text instead of an image.

AndrewRadev committed:
Fix code snippet

AndrewRadev committed:
Use text for the clipboard button, make it a link

This makes it more consistent with the other "button", and easier to
style correctly.

AndrewRadev committed:
Apply clipboard button changes to free tasks
